### DISCUSSION TITLE
Show your notes only

### DISCUSSION THREAD

**Discussion by SnivyFan16** [JordanSantiagoYT/FNF-JS-Engine#431] [Ideas]
This might be a stupid idea, but i just personally think it would be nice to have, so I had an idea for an option (or suboption if you will) where it does basically the same thing as the "Show Notes" option, but it only prevents the opponents notes from getting rendered, while your notes still show up

I got this idea cause of "Imprisonment", where bf's side is actually playable (and kinda fun imo), but bambi's side reaches ridiculous NPS, and always wanted to play the full chart (without deleting bambi's side myself) but couldnt cause the game cant handle that many notes at once for me (Show notes is off for this screenshot btw) (and yes I do know that the reason for all those notes is cause of an increased note multiplier)
![image_2024-07-11_133433782](https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/175345812/add33619-9014-49fe-b281-5303d214c4fe)

Now im not a coder or anything, so idk if telling the game to see what notes are on the opponents side and not render them is actually all that hard. :P

**Comment by moxie-coder** [comment]
there’s already an option for that you fucking moron

**Reply by SnivyFan16** [reply]
You could have just politely told me that it already exists and what it was called instead of insulting me but ok then… (Also the feature you are likely talking about (Opponents notes in gameplay settings) still renders the opponents notes, all it does to them is basically turn the opacity to 0 while still rendering them likely due to it also rendering bf's notes, so there technically isnt already an option lmao)
